client/me/purchases/manage-purchase/purchase-meta-expiration.tsx

```diff
@@ -66,13 +68,13 @@ function PurchaseMetaExpiration( {
 	if ( isRenewable( purchase ) && ! isExpired( purchase ) ) {
 		const dateSpan = <span className="manage-purchase__detail-date-span" />;
 		// If a jetpack site has been disconnected, the "site" prop will be null here.
-		const shouldRenderToggle = site && isProductOwner;
+		const shouldRenderToggle = ( isCancellableSitelessPurchase || site ) && isProductOwner;
 
 		const autoRenewToggle = shouldRenderToggle ? (
 			<AutoRenewToggle
-				planName={ site.plan?.product_name_short }
-				siteDomain={ site.domain }
-				siteSlug={ site.slug }
+				planName={ isCancellableSitelessPurchase ? '' : site.plan?.product_name_short }
+				siteDomain={ isCancellableSitelessPurchase ? '' : site.domain }
+				siteSlug={ isCancellableSitelessPurchase ? '' : site.slug }
```

Comment:
It seems like this change is breaking some unit tests.

Reply:
Ah - thanks, getting that fixed 👍